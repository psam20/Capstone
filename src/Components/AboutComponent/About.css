h3{
    color: blue;
    font-weight: bold;
    padding-left: 20px;
    padding-top: 20px;
}

p{
    color: brown;
    padding-left: 20px;
    padding-right: 20px;
    text-align: justify;
}