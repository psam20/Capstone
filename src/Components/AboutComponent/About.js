import React from 'react';
import './About.css';
import 'bootstrap/dist/css/bootstrap.min.css'

class About extends React.Component{
    constructor(){
        super();
        this.state={
            title: "About Us"
        }
    }
    render(){
        return(
            <div>
                <h3 className="head">{this.state.title}</h3>
                <p className="para">
                    Shopify is an e-commerce company based in Bengaluru, Karnataka, India. 
                    It was founded in 2007. The company initially focused on book sales, 
                    before expanding into other product categories such as consumer 
                    electronics, fashion, home essentials & groceries, and lifestyle 
                    products.
                </p>
                <p className="para">
                    Shopify.com's product lines available at its website include several 
                    media (books, DVDs, music CDs, videotapes and software), apparel, baby 
                    products, consumer electronics, beauty products, gourmet food, groceries, 
                    health and personal-care items, industrial & scientific supplies, 
                    kitchen items, jewelry, watches, lawn and garden items, musical 
                    instruments, sporting goods, tools, automotive items and toys & games.
                    In August 2019, Shopify applied to have a liquor store in San Francisco,
                    CA as a means to ship beer and alcohol within the city. Shopify has 
                    separate retail websites for some countries and also offers international 
                    shipping of some of its products to certain other countries.
                </p>
                </div>
             )
    }
} 

export default About;