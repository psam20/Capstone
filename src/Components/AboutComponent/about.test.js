import React from 'react';
import {shallow} from 'enzyme';

import About from './About';

describe('About Page Testing',()=>{
    let Wrapper=shallow(<About />);
     it('Should Render About Page Correctly',()=>{
         expect(Wrapper).toMatchSnapshot();
     })
})
