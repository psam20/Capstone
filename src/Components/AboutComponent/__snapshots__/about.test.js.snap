// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Testing Should Render About Page Correctly 1`] = `ShallowWrapper {}`;
